Datastore/src/DatastoreClient.php

```diff
@@ -330,7 +330,8 @@ public function keys($kind, array $options = [])
      * use Google\Cloud\Datastore\Entity;
      *
      * class OtherPersonEntity extends Entity
-     * {}
+     * {
+     * }
```

Comment:
Because phpdocumentor uses `{}` as a token, it was replacing this with `}`, which resulted in a syntax error. The solution (which is more readable anyway) is to split this into separate lines.